classes/controller/AdminController.php

```diff
@@ -1367,6 +1367,11 @@ protected function processUpdateOptions()
     {
         $this->beforeUpdateOptions();
```

Comment:
shouldn't the hook be executed inside the `beforeUpdateOptions` function

Reply:
None of the classes extending AdminController and declaring this method ([list](https://github.com/PrestaShop/PrestaShop/search?utf8=%E2%9C%93&q=function+beforeUpdateOptions&type=)) are currently executing `parent::beforeUpdateOptions()`.